DALK.PL_ANALYZER/Views/Matches/Index.cshtml

```diff
@@ -3,41 +3,77 @@
 @{
     ViewBag.Title = "Meczyki";
```

Comment:
Please drop all usage of ViewBag/TempData its always a bad idea

Reply:
DONE.